ID.me WebVerify SDK/IDmeWebVerify.m

```diff
@@ -304,6 +308,39 @@ - (void)getAccessTokenWithScope:(NSString* _Nullable)scope forceRefreshing:(BOOL
 
 }
 
+- (void)refreshTokenForScope:(NSString* _Nonnull)scope callback:(IDmeVerifyWebVerifyTokenResults _Nonnull)callback {
```

Comment:
I do not think so. This method is just called internally and only if there is a scope (either passed to `getAccesToken` or  the latest used scope from the keychain)